Services/UserService/Validators/ErrorsMessages.resx

```diff
@@ -0,0 +1,159 @@
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="BirthdatEmpty" xml:space="preserve">
+    <value>Birthday must not be empty.</value>
+  </data>
+  <data name="BirthdayYoung" xml:space="preserve">
+    <value>You are too young</value>
+  </data>
+  <data name="EmailEmpty" xml:space="preserve">
+    <value>Email must not be empty.</value>
+  </data>
+  <data name="EmailTooLong" xml:space="preserve">
+    <value>Login length up to 50 characters</value>
+  </data>
+  <data name="EmptyPassword" xml:space="preserve">
+    <value>Password must not be empty.</value>
+  </data>
+  <data name="FirstNameEmpty" xml:space="preserve">
+    <value>First name must not be empty.</value>
+  </data>
+  <data name="FirstNameError" xml:space="preserve">
+    <value>The first letter must be uppercase and the rest must be lowercase. First name must contain only letters</value>
```

Comment:
Name is too short is also checks here

Reply:
Names and surnames can consist of 1 letter
https://white-fly.diary.ru/p39835648.htm?oam

Reply:
Fixed